InterviewFlashCards/IFCFlashCard.m

```diff
@@ -13,18 +12,16 @@ - (instancetype)initWithDictionary:(NSDictionary *)dict {
         self.answer           = dict[@"answer"];
         self.questionImageURL = dict[@"question_url"];
 
-        NSDictionary *urls    = dict[@"answer_urls"];
-        self.answerImageURLs = [self arrayOfUrlsFromDictionary:urls];
-
-        if (self.questionImageURL) {
-            self.questionImages = [NSMutableArray new];
-        }
-        if (self.answerImageURLs) {
-            self.answerImages = [NSMutableArray new];
+        id urls    = dict[@"answer_urls"];
+        if ([urls isKindOfClass:[NSArray class]]) {
+            self.answerImageURLs = urls;
+        } else if ([urls isKindOfClass:[NSDictionary class]]) {
```

Comment:
This was added (as gross as it is 😷) to support the firebase DataGenerator in case we went backwards. I'd be happy to drop it though